misc/multiaddr/src/lib.rs

```diff
@@ -59,6 +60,11 @@ impl Multiaddr {
         self.bytes.len()
     }
 
+    /// Returns true if the length of this multiaddress is 0.
+    pub fn is_empty(&self) -> bool {
```

Comment:
Was this a clippy suggestion? Is it used anywhere?

Reply:
Clippy warns if a type has a `len()` method but no `is_empty()` method.